common/src/main/java/software/amazon/documentdb/jdbc/common/utilities/JdbcType.java

```diff
@@ -21,6 +21,11 @@
 
 /**
  * Copy of the java.sql.Types constants but as an enum, for use in lookups.
+ * Warning: if a JDBC type is added or deprecated, the change should be reflected
+ * on the ODBC driver as well. Files to be changed in the ODBC driver:
+ * src\binary\include\ignite\impl\binary\binary_common.h, and
```

Comment:
Let's not include any paths - this will because out of date. Just file name and preferably the class name or namespace.